tests/transformers/test_transformers_model_export.py

```diff
@@ -3633,3 +3633,14 @@ def test_peft_pipeline_copy_metadata_in_databricks(mock_is_in_databricks, peft_p
     else:
         assert not os.path.exists(metadata_path)
     mock_is_in_databricks.assert_called_once()
+
+
+@pytest.mark.parametrize("device", ["cpu", "cuda", 0, -1, None])
```

Comment:
Just out of curiosity, is `-1` a valid value for `device`?

Reply:
Yes https://github.com/mlflow/mlflow/blob/3267c372f2dbb98d5f5870f3d7262f5b7591ecd0/mlflow/transformers/__init__.py#L149-L152